### DISCUSSION TITLE
WebDav 无法挂载

### DISCUSSION THREAD

**Discussion by YangguangZhou** [AlistGo/alist#4281] [Q&A]
按照文档进行配置后，WebDav 依然无法挂载，显示端口关闭
![Screenshot_20230430-151400.png](https://user-images.githubusercontent.com/61733195/235341530-519b195d-5512-44a5-92ef-b017a9522ca4.png)

![Screenshot_20230430-151725.png](https://user-images.githubusercontent.com/61733195/235341529-99c5376a-5aa6-40f9-a20b-a017c8634a01.png)

![](https://user-images.githubusercontent.com/36558727/235341447-f471f2d7-78b7-4f68-9962-4c36b2e0be15.png)

**Comment by YangguangZhou** [comment]
@Xhofe

**Comment by xhofe** [comment]
请自查服务状态及端口开放情况。

**Comment by YangguangZhou** [comment]
443 和 80 端口是开放的
可是依然失败
![Screenshot_20230430-155527.png](https://user-images.githubusercontent.com/61733195/235342293-41039b59-79f7-403a-a745-470936d4f8ea.png)
![Screenshot_20230430-155633.png](https://user-images.githubusercontent.com/61733195/235342302-fd064184-744c-4977-96bb-8e55e07608f9.png)

**Comment by anwen-anyi** [comment]
感觉是玄学问题
我两个 内网自用的 和 一个公网服务器的
 - 先说说内网的
   - 内网A 安卓端 和 ISO 端都能挂载
   - 内网B 安卓端 不能挂载，IOS端可以挂载
 - 公网服务器
   - 安卓端 不能挂载，IOS端可以
-----

我也一脸懵逼，而且内网B之前是可以挂载的 不知道哪个版本 升级后 不能挂载了，开始是安卓能，IOS不能的，现在完全反过来了 :joy: :joy:
